src/components/Menu/LanguageNavigation/LanguageNavigation.tsx

```diff
@@ -37,18 +37,23 @@ export interface LanguageNavigationProps {
   setPreviousSDKInterfaceTab: Dispatch<SetStateAction<string>>;
 }
 
-const changePageOnSelect = (pageLanguage: string) => (newValue: SingleValue<ReactSelectOption>) => {
-  if (newValue) {
-    const language = newValue.value;
-    const { isLanguageDefault, isPageLanguageDefault } = getLanguageDefaults(
-      getTrimmedLanguage(language),
-      pageLanguage,
-    );
+const changePageOnSelect =
+  (pageLanguage: string, cb: (arg: string) => void) => (newValue: SingleValue<ReactSelectOption>) => {
```

Comment:
Could a more descriptive name be considered for `cb` here? I'm assuming it means "callback" but that's not certain

Reply:
Oh yeah, for sure it is a callback function, would you like me to change it to callback ?